tools/pypi_rss/main.go

```diff
@@ -0,0 +1,224 @@
+package main
```

Comment:
could we add a brief comment talking about what this binary does and pointing to the pypi RSS feed docs?